Components/Input/input.css

```diff
@@ -0,0 +1,58 @@
+@import url(/CSS/variable.css);
+.form-container {
+    background-color: var(--prime-blue);
+    font-family: 'Open Sans', sans-serif;
+    padding: 2rem;
+    border-radius: 4px;
+    width: 300px;
+}
+
+.header {
+    padding: 1rem 2rem 1rem 0rem;
+    font-size: x-large;
+}
+
+.form-control input {
+    display: block;
+    border: 3px solid transparent;
+    padding: 0.7rem;
+    width: 100%;
+}
+
+.form-control label {
+    font-size: small;
+    display: inline-block;
+    margin: 15px 0px 5px 0px;
```

Comment:
you should always prefer rem instead of px for better web performence

Reply:
Oh okay Thankyou, will make the change